assets/js/script.js

```diff
@@ -37,6 +38,11 @@ function findTarot() {
             console.log(randomIndex);
             const randomCard = cardsArr[randomIndex];
             console.log(randomCard);
+            const randomPicture = pictureArr[randomIndex];
```

Comment:
was going to generate from array - leaving this here to remind me of my backup plan if my setattribute idea doesnt work out :)